src/theme/base/typography.ts

```diff
@@ -66,6 +66,45 @@ const typography = {
   //       italic: 'Roboto-BoldItalic',
   //     },
   //   },
+
+  //   Montserrat: {
```

Comment:
Are these comments required?

Reply:
Yes, actually we can remove it but kept it to make it easier to test custom fonts. 
We're loading Montserrat in example app. Just uncommenting will make it work here. Let me know if needs to be removed!